src/core/server/services/comments/pipeline/phases/index.ts

```diff
@@ -61,9 +61,9 @@ export const moderationPhases: IntermediateModerationPhase[] = [
   detectLinks,
 
   // Apply any pre-existing conditions to these comments.
+  statusPreModerateNewCommenter,
```

Comment:
Moved up so that it will tag "new commenter" before it gets to the "site wide premod" check.

If site wide pre-mod is enabled, it will fall through to the site-wide pre-mod anyways, so this order doesn't hurt anything, it just allows for an earlier pre-mod check based on new commenter state prior to any site-wide pre-mod checks.